### DISCUSSION TITLE
Retiarii_example_multi-trial_NAS

### DISCUSSION THREAD

**Discussion by sanjaykumarkhadka** [microsoft/nni#3850] [Q&A]
i am using latest NNI(2.3 version) on jupyter notebook. i am not able to execute the example provided in the documentation on github.

https://github.com/microsoft/nni/blob/master/examples/notebooks/Retiarii_example_multi-trial_NAS.ipynb

I am finding the error @model_wrapper decorator 
![Screenshot 2021-06-19 at 20 41 51](https://user-images.githubusercontent.com/33345291/122652361-c9dfd880-d13e-11eb-8370-1304063ba068.png)

please let me know if i am doing something wrong.

**Comment by ultmaster** [comment]
Please replace `super(Net, self).__init__()` with `super().__init__()` and see if that works?

**Reply by sanjaykumarkhadka** [reply]
It dint work,

AttributeError: 'NoneType' object has no attribute '__name__'

**Reply by kvartet** [reply]
hello @sanjaykumarkhadka, you can pull the master branch and try again.

**Reply by sanjaykumarkhadka** [reply]
Hi, I am still getting error 

RuntimeError: tensor op type aten::view has more than one matched: [[('size', 'List[int]', 'None')], [('dtype', 'int', 'None')]]

**Reply by kvartet** [reply]
Which version of your PyTorch? we have only tested with PyTorch 1.6 and 1.7.

**Comment by ultmaster** [comment]
Would you try the latest example. The example is updated. Also, PyTorch version >= 1.8 is not supported for now.
